mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -190,6 +190,11 @@ def test_subject_info():
         assert_equal(subject_info[key], raw_read.info['subject_info'][key])
     assert_equal(raw.info['meas_date'], raw_read.info['meas_date'])
 
+    for key in ['secs', 'usecs', 'version']:
+        assert_equal(raw.info['meas_id'][key], raw_read.info['meas_id'][key])
```

Comment:
FYI you can now do the simpler `assert raw.info['meas_id'][key] == raw_read.info['meas_id'][key]`. The codebase is slowly migrating toward this `pytest` way of doing things (except for comparisons that require `numpy.testing`). Feel free to change it here or not, I'm happy either way